.gitignore

```diff
@@ -64,6 +64,9 @@ typings/
 
 # Output of 'npm pack'
 *.tgz
+!packages/ni-eslint-config-angular-3.4.0.tgz
```

Comment:
Placeholder comment to block submission until you've swapped to published packages.